standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1031,6 +1031,9 @@ public static ConfVars getMetaConf(String name) {
     REPL_METRICS_MAX_AGE("metastore.repl.metrics.max.age",
       "hive.metastore.repl.metrics.max.age", 7, TimeUnit.DAYS,
       "Maximal age of a replication metrics entry before it is removed."),
+    REPL_TXN_TIMEOUT("metastore.repl.txn.timeout", "hive.repl.txn.timeout", 1, TimeUnit.DAYS,
```

Comment:
The default value of this should be a very high value.

Reply:
Thanks for taking a look! What default value do you suggest?

Reply:
REPL_TXN_TIMEOUT  is the time after hive.repl.event.db.listener.timetolive after which replication transactions are timed out. So the real timeout is (hive.repl.event.db.listener.timetolive  + REPL_TXN_TIMEOUT).